optaplanner-core/src/main/java/org/optaplanner/core/impl/localsearch/decider/acceptor/greatdeluge/GreatDelugeAcceptor.java

```diff
@@ -0,0 +1,118 @@
+package org.optaplanner.core.impl.localsearch.decider.acceptor.greatdeluge;
+
+import org.optaplanner.core.api.score.Score;
+import org.optaplanner.core.impl.localsearch.decider.acceptor.AbstractAcceptor;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchMoveScope;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchPhaseScope;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchStepScope;
+import org.optaplanner.core.impl.score.ScoreUtils;
+
+public class GreatDelugeAcceptor extends AbstractAcceptor {
+
+    private Score initialLevel = null;
```

Comment:
the term "level" is used both by as score level and as water level. That creates ambiguity/confusion.
Powerthesaurus didn't help though, so I don't have a good alternative.
I did find "gauge height". Maybe we can use that?